### PR TITLE
chore(docs): updating the implementation of the theme switch button

### PR DESCRIPTION
**Pull request by yj-liuzepeng**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

## Description

image

[docs/.vitepress/vitepress/components/common/vp-switch.vue](https://github.com/element-plus/element-plus/blob/dev/docs/.vitepress/vitepress/components/common/vp-switch.vue)

Now that the switch supports[ custom action icon](https://element-plus.gitee.io/en-US/component/switch.html#custom-action-icon), we can optimize the implementation of the theme switch button.

### CODE REVIEW

File: docs/.vitepress/vitepress/components/icons/dark.vue [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
   </svg>
 </template>
+
+<style scoped lang="scss">
+.svg__icon {
+  border-radius: 50%;
+  color: #cfd3dc;
+  background-color: #141414;
+}
+</style>
```
Comment (btea) [inline docs/.vitepress/vitepress/components/icons/dark.vue:16]: The styles here may all be put into `common/vp-theme-toggler.vue` component uniformly.

File: docs/.vitepress/vitepress/components/full-screen/vp-theme-toggler.vue [hunk 2]
```diff
@@ -9,11 +9,13 @@ const locale = useNavbarLocale()
...
-      :aria-checked="isDark"
-      @click="toggleDark()"
-    />
+    <ClientOnly>
+      <CommonThemeToggler
+        :aria-label="locale['theme-toggler']"
+        :aria-checked="isDark"
+        @click="toggleDark()"
+      />
+    </ClientOnly>
```
Comment (btea) [inline docs/.vitepress/vitepress/components/full-screen/vp-theme-toggler.vue:18]: Is `clientOnly` necessary here? 🤔

Comment (yj-liuzepeng) [reply docs/.vitepress/vitepress/components/full-screen/vp-theme-toggler.vue]: I made some modifications. Please take a look to see if this approach might be better?

Comment (yj-liuzepeng) [reply docs/.vitepress/vitepress/components/full-screen/vp-theme-toggler.vue]: > Is `clientOnly` necessary here? 🤔

isDark relies on the native browser API, so I wrapped it with `ClientOnly`

**Review by btea — APPROVED** [review]
Thanks!

**Pull request merged**